_opensearch/rest-api/nodes-apis/nodes-hot-threads.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: default
+title: Nodes hot threads
+parent: Nodes APIs
+grand_parent: REST API reference
+nav_order: 10
+---
+
+# Nodes hot threads
+
+This REST API provide information about busy JVM threads for selected cluster nodes.
+It provides a unique view of what activity each node spends time on.
```

Comment:
maybe instead of "...view of what activity each node spends time on. " 
It provides a unique view of "the activities in which each node is engaged."